[2025-09-26T17:08:21.753+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-26T17:08:21.794+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mon_premier_dag.task_2 scheduled__2025-09-17T00:00:00+00:00 [queued]>
[2025-09-26T17:08:21.821+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mon_premier_dag.task_2 scheduled__2025-09-17T00:00:00+00:00 [queued]>
[2025-09-26T17:08:21.823+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-09-26T17:08:21.862+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): task_2> on 2025-09-17 00:00:00+00:00
[2025-09-26T17:08:21.878+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'mon_premier_dag', 'task_2', 'scheduled__2025-09-17T00:00:00+00:00', '--job-id', '183', '--raw', '--subdir', 'DAGS_FOLDER/mon_premier_dag.py', '--cfg-path', '/tmp/tmpck1hytgw']
[2025-09-26T17:08:21.891+0000] {standard_task_runner.py:105} INFO - Job 183: Subtask task_2
[2025-09-26T17:08:21.892+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=267) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-26T17:08:21.897+0000] {standard_task_runner.py:72} INFO - Started process 271 to run task
[2025-09-26T17:08:21.995+0000] {task_command.py:467} INFO - Running <TaskInstance: mon_premier_dag.task_2 scheduled__2025-09-17T00:00:00+00:00 [running]> on host 4461bcf72552
[2025-09-26T17:08:22.178+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mon_premier_dag' AIRFLOW_CTX_TASK_ID='task_2' AIRFLOW_CTX_EXECUTION_DATE='2025-09-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-17T00:00:00+00:00'
[2025-09-26T17:08:22.180+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-26T17:08:22.204+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-09-26T17:08:22.206+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "This is my first DAG!"']
[2025-09-26T17:08:22.226+0000] {subprocess.py:99} INFO - Output:
[2025-09-26T17:08:22.230+0000] {subprocess.py:106} INFO - This is my first DAG!
[2025-09-26T17:08:22.232+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-09-26T17:08:22.274+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-26T17:08:22.276+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=mon_premier_dag, task_id=task_2, run_id=scheduled__2025-09-17T00:00:00+00:00, execution_date=20250917T000000, start_date=20250926T170821, end_date=20250926T170822
[2025-09-26T17:08:22.318+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-26T17:08:22.351+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-26T17:08:22.355+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
